src/assets/index.ts

```diff
@@ -233,36 +270,45 @@ const weth_ethereum: AssetConfig = {
   chain_aliases: {
     axelar: {
       assetSymbol: "WETH",
-      assetName: "WETH (Axelar-wrapped)",
-      minDepositAmt: 0.1,
+      assetName: "WETH",
+      minDepositAmt: 2e-05
```

Comment:
The python JSON converter uses scientific notation for some reason after enough decimals, and the fix for it is convoluted. If the frontend still shows it as a normal float instead, then we can leave it as is.